spec/setup_test_database.rb

```diff
@@ -0,0 +1,9 @@
+require 'pg'
+
+p 'Setting up database...'
+
+def setup_test_database
+  connection = PG.connect(dbname: 'chitter_test')
```

Comment:
Could we have created a DatabaseConnection class and then call the class each time so you don't have to repeat the PG.connect process?